static/app/views/alerts/rules/metric/details/sidebar.tsx

```diff
@@ -76,6 +76,7 @@ function TriggerDescription({
     AlertWizardAlertNames[getAlertTypeFromAggregateDataset(rule)]
   );
 
+  // TODO: update this text if rule type is dynamic
```

Comment:
Was this addressed or is this a future TODO?

Reply:
This was addressed, removing